pyspecProcScripts/correlation_alignment.py

```diff
@@ -52,8 +61,23 @@ def correl_align(s, align_phases=False,tol=1e-4,indirect_dim='indirect',fig_titl
                 the width of the Gaussian function used to frequency filter
                 the data in the calculation of the correlation function.
     """
-    ph1_len = len(s.getaxis('ph1'))
-    ph2_len = len(s.getaxis('ph2'))
+    if 'nScans' in s.dimlabels:
+        phcycdims = [j for j in s.dimlabels if j.startswith('ph')]
+        indirect = set(s.dimlabels)-set(phcycdims)-set([direct])
+        indirect = [j for j in s.dimlabels if j in indirect]
+        nScans_len = len(s.getaxis('nScans'))
+        s.smoosh(indirect)
+    if 'ph2' in s.dimlabels:
+        ODNP=False
```

Comment:
when IR is used there is no ph2 dimension as we only phase cycle the 90 pulse

Reply:
no, this is not the way to go -- rather you should pass your coherence pathway as a dictionary